File: solutions/uri/1160/1160.cpp
#include <cstdio>
#include <iostream>

int main() {
    int n, c;
    int a, b;
    float ac, bc;

    std::cin >> n;
    while (n--) {
        c = 0;
        scanf("%d %d %f %f", &a, &b, &ac, &bc);
        while (a <= b) {
            a *= (ac / 100.0) + 1.0;
            b *= (bc / 100.0) + 1.0;
            c++;
            if (c > 100) {
                printf("Mais de 1 seculo.\n");
                break;
            }
        }
        if (c <= 100) {
            printf("%d anos.\n", c);
        }
    }
    return 0;
}

File: solutions/beecrowd/1160/1160.py
t = int(input())

for _ in range(t):
    pa, pb, g1, g2 = input().split()
    pa, pb = map(int, [pa, pb])
    g1, g2 = map(float, [g1, g2])
    count_years = 0

    while pa <= pb:
        pa = int(pa * ((g1 / 100.0) + 1.0))
        pb = int(pb * ((g2 / 100.0) + 1.0))
        count_years += 1

        if count_years > 100:
            print('Mais de 1 seculo.')
            break

    if count_years <= 100:
        print(f'{count_years} anos.')
